classes/ClaytonTestClass3.cls

```diff
@@ -0,0 +1,16 @@
+public class ClaytonTestClass3 {
+    public void AWrongMethodN_a_me_incredible_wow(){
+        Boolean WrongFormat_VARIABLE = false;
+    }
+    
+    public void anotherMethod(){
+        for(Contact cnt : [Select Id, AccountId From Contact]){
```

Comment:
If the table in this query contains more than 50'000 records a runtime exception will be thrown. Consider either introducing a `WHERE` clause, or adding a `LIMIT` to contain the number of records returned.